corehq/apps/domain/models.py

```diff
@@ -913,6 +913,16 @@ def copies_of_parent(self):
         return Domain.view('domain/copied_from_snapshot', keys=[s._id for s in self.copied_from.snapshots()], include_docs=True)
 
     def delete(self):
+        from corehq.apps.domain.signals import commcare_domain_pre_delete
+
+        results = commcare_domain_pre_delete.send_robust(sender='domain', domain=self)
+        for result in results:
+            if result[1]:
+                notify_exception(
```

Comment:
this is only accessible from the command line, right? failing hard is probably fine for that scenario